src/app/features/post/post.component.html

```diff
@@ -21,7 +21,12 @@ <h2>{{ post.excerpt }}</h2>
         </span>
         <div class="post__header-authors">
           @for (author of post.authors; track $index) {
-            <blog-author [author]="author" size="lg"></blog-author>
+            <a
+              [attr.data-role]="$any(author).url ? 'button' : ''"
+              [routerLink]="'/' + $any(author).url"
```

Comment:
Do we need this `$any`?
```suggestion
              [routerLink]="'/' + author?.url"
```

Reply:
I need it. I have an error
<img width="539" alt="image" src="https://github.com/Backbase/backbase.github.io/assets/28485877/26955931-f875-4bbe-a442-838d74b851cf">
<img width="453" alt="image" src="https://github.com/Backbase/backbase.github.io/assets/28485877/0536cdab-9059-49f4-b24b-bb23b8dff8c4">

Reply:
I can create some pipe or method instead of `$any`.